source/scripts/constants.js

```diff
@@ -0,0 +1,3 @@
+export const API_KEY = '857b051abfmsh018f07c4f390b41p1547b1jsn64ae2598745b';// prevent exposing api key
```

Comment:
Perhaps renaming this to config.js would make it a little more obvious what it means.